src/parse.jl

```diff
@@ -7,7 +7,7 @@ import ..Util.log
 
 export well_formed, matched_rule, parse_command, synonym, partial_well_formed
 
-REGS = (
+const REGS = (
```

Comment:
ya know as a Java developer by trade I'm embarrassed to have so many un-`const`ed fields!